phylogenetic/config/229e/auspice_config.json

```diff
@@ -19,19 +22,18 @@
       "key": "country",
       "title": "country",
       "type": "categorical"
-    },
-    {
-      "key": "host",
-      "title": "host",
-      "type": "categorical"
     }
   ],
+  "geo_resolutions": [
+    "country",
+    "region"
```

Comment:
_non-blocking_

Slightly weird to me that `region` is available as a geo_resolution but not as a coloring or filter. I think it would match other Nextstrain pathogen builds (e.g. [zika](https://next.nextstrain.org/zika?c=region) or [measles](https://next.nextstrain.org/measles/genome?c=region)) to include `region` as a coloring.

Reply:
good call on being consistent with prior art — updated.